vspec/vssexporters/vss2binary.py

```diff
@@ -130,18 +121,19 @@ def export(config: argparse.Namespace, root: VSSNode, print_uuid):
     global _cbinary
     dllName = "../../binary/binarytool.so"
     dllAbsPath = os.path.dirname(os.path.abspath(__file__)) + os.path.sep + dllName
+    if not os.path.isfile(dllAbsPath):
```

Comment:
This is the functional change - give a user friendly error if library does not exist. The error is expected if you download by pip and do `vspec2x.py -format binary` as you then have no library